lib/logto_client.dart

```diff
@@ -6,11 +6,12 @@ import 'package:jose/jose.dart';
 import '/logto_core.dart' as logto_core;
 import '/src/exceptions/logto_auth_exceptions.dart';
 import '/src/interfaces/logto_interfaces.dart';
+import 'src/modules/id_token.dart';
```

Comment:
just curious, what is the difference between `/src` and `src`

Reply:
hmm good call, should be a typo but looks like the leading / is optional.

Reply:
updated